src/vue/modules/dividend-form/index.vue

```diff
@@ -0,0 +1,530 @@
+              white-autofill
+              class="app__form-field"
+              v-model.trim="form.amount"
+              type="number"
+              name="dividend-amount"
+              :step="config.minAmount"
+              @blur="touchField('form.amount')"
+              :label="'dividend-form.amount' | globalize({
+                asset: form.asset.code
+              })"
+              :disabled="formMixin.isDisabled"
+              :error-message="getFieldErrorMessage('form.amount', {
+                available: form.asset.balance.value,
+                maxDecimalDigitsCount: config.decimalPoints,
+              })"
+            />
+          </div>
+
+          <div class="app__form-row dividend__form-row">
+            <table class="dividend__fee-table">
+              <thead class="dividend__fee-thead">
+                <tr>
+                  <td>
+                    {{ 'dividend-form.email' | globalize }}
+                  </td>
+                  <td>
+                    {{ 'dividend-form.token-amount' | globalize }}
+                  </td>
+                  <td>
+                    {{ 'dividend-form.supposed-dividend-amount' | globalize }}
+                  </td>
+                </tr>
+              </thead>
+              <tbody
+                class="dividend__fee-tbody"
+                :class="{ 'dividend__data--loading': isSignersLoadPending }"
+                v-if="balanceHolders.length && isSignersLoaded"
+              >
+                <tr v-for="holder in balanceHolders" :key="holder.id">
+                  <td>
+                    <email-getter :balance-id="holder.id" />
+                  </td>
+                  <td>
+                    {{ holder.available | formatMoney }}
+                    {{ form.ownedAsset.code }}
+                  </td>
+                  <td v-if="form.amount">
+                    {{ getDividendAmount(holder) | formatMoney }}
+                    {{ form.asset.code }}
+                  </td>
+                  <td v-else>
+                    &mdash;
+                  </td>
+                </tr>
+              </tbody>
+            </table>
+            <template v-if="!balanceHolders.length && isSignersLoaded">
+              <p class="dividend__fee-holders-not-found">
+                {{ 'dividend-form.asset-holders-not-found' | globalize }}
+              </p>
+            </template>
+          </div>
+
+          <div class="app__form-actions dividend__action">
+            <button
+              v-ripple
+              v-if="!formMixin.isConfirmationShown"
+              type="submit"
+              class="app__button-raised"
+              :disabled="formMixin.isDisabled || !balanceHolders.length"
+              form="dividend-form"
+            >
+              {{ 'dividend-form.dividend-btn' | globalize }}
+            </button>
+            <form-confirmation
+              v-if="formMixin.isConfirmationShown"
+              @ok="hideConfirmation() || submit()"
```

Comment:
Move `hideConfirmation()` to the end of `submit` method. Also, you need to define a new variable that will tell to component about form submitting and pass it here via `isPending`. Look how this prop works for formConfirmation component